llvm/lib/Target/X86/X86FixupVectorConstants.cpp

```diff
@@ -263,6 +290,34 @@ bool X86FixupVectorConstantsPass::processInstruction(MachineFunction &MF,
     return false;
   };
 
+  auto ConvertToZeroUpper = [&](unsigned OpUpper64, unsigned OpUpper32,
+                                unsigned OperandNo) {
```

Comment:
Do we need this argument? I see we always pass `1` to it.

Reply:
Not really, unless we ever have the need to support avx512 predicate move loads?